src/arbitrary/primitives.rs

```diff
@@ -0,0 +1,41 @@
+//-
+// Copyright 2017, 2018 The proptest developers
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+//! Arbitrary implementations for primitive types.
+
+use bool;
+use char;
+use num::{isize, usize, f32, f64, i16, i32, i64, i8, u16, u32, u64, u8};
+
+arbitrary!(
+    bool, f32, f64,
```

Comment:
For primitives other than floating points I think the whole input space is right (unlike Haskell QC which uses `0..100`, and is in the wrong here if you ask me..), but for floats it could make more sense to limit the space. I think `POSITIVE | NEGATIVE | ZERO | SUBNORMAL | NORMAL` sounds reasonable. I'll change to this for now.